visualqa/trainMLP.py

```diff
@@ -95,33 +106,81 @@ def main():
     data_root = args.dataroot
     experiment_root = args.experiment_root
 
+    # Set both the numpy and the Python random seeds.
+    random.seed(RANDOM_SEED)
+    np.random.seed(RANDOM_SEED)
```

Comment:
How does weight initialization in Keras work? I guess we also need to set a seed?